packages/block-library/src/pullquote/save.js

```diff
@@ -9,8 +9,10 @@ import clsx from 'clsx';
 import { RichText, useBlockProps } from '@wordpress/block-editor';
 
 export default function save( { attributes } ) {
-	const { textAlign, citation, value } = attributes;
-	const shouldShowCitation = ! RichText.isEmpty( citation );
+	const { textAlign, citation, value, metadata } = attributes;
+	const hasBoundCitation = !! metadata?.bindings?.citation;
+	const shouldShowCitation =
+		hasBoundCitation || ! RichText.isEmpty( citation );
```

Comment:
@cbravobernal, circling back to our chat earlier this week, this check allows us not to render the `cite` tag when the binding is not set, making the `save` function fully backward compatible.